LAVA_setup_guide.md

```diff
@@ -0,0 +1,172 @@
+
+`sudo mkdir -p /opt/lava-worker`
+`sudo chmod 777 /opt/lava-worker`
+`cd /opt/lava-worker`
+`git clone https://github.com/paul-szczepanek-arm/lava-prototype lava-scripts`
+
+## Running LAVA server docker compose
+
+Instead of installing your lava server you may use docker compose provided by us.
+
+- `git clone https://github.com/ARMmbed/mbed-os-lava-docker-compose`
+-  Create a certificate and key and replace the ones in apache2/cert (you can try the free cert from
+   [zerossl](https://app.zerossl.com/certificate/new))
+- `docker-compose build lava-server`
+- `docker-compose up lava-server`
+
+Website is available on localhost, username and passowrd are `lava-admin`. Change the password after logging in.
+
+## Running a premade SD card image with the worker on your RPi
+
+Download the SD card image [MISSING LINK](http://) and copy it direclty to the root of the sd card device with `dd`.
+
+You will need to edit the config file to give the worker its unique name and password. If you're using a different
+server you will also need to change the server URL.
+
+`sudo vim /etc/lava-dispatcher/lava-worker`
+    * set WORKER_NAME (and URL if needed)
+    * set the token to the string you get from adding a worker in the server
+
+## Adding Device types in the server
+
+Open the web interface of the lava server and enter administration tab.
+
+- In "Home › lava_scheduler_app › Devices" add a new device with the same name as the file (mbed_nrf or mbed_disco)
+- Select device type mbed (click plus to add new device type, name it docker, untick the health check or leave ticked
+  but then you need the health check below)
+- Add worker (only hostname matters and has to match what is in the worker file in that worker, for example RPI001)
+- The token string has to match the string in the worker file on that worker machine.
+- Set health of device to unknown
+- Save device
+
+## Running tests
+
+Tests are run by submitting jobs on the server through the web interface after logging in or with
+[lavacli](https://docs.lavasoftware.org/lava/lavacli.html). The server will wait until a worker connects to it
+and it will dispatch the job to a matching device hosted on one of such workers. Hence only the server ip needs to be
+known.
+
+### Network environment
+
+RPis need to be on the outside network for security reasons. For network tests you'll need two APs. One unsecured with
+SSID: UNSECURED. One secured with password: PASSWORD and SSID: SECURED.
+
+### Verification
+
+You can submit a test job to verify it works. Connect a NRF52840_DK to your worker and submit one of the jobs
+(http://localhost/scheduler/jobsubmit) in https://github.com/paul-szczepanek-arm/lava-prototype/tree/master/jobs
+
+### Healthcheck
+
+Healthcheck is now part of the lava prototype repo. It's a job that runs every day and makes sure the board flashes and connects.
+
+https://github.com/paul-szczepanek-arm/lava-prototype/tree/master/lava-server/dispatcher-config/health-checks
```

Comment:
Can you update the link ? 
How do one create a healthcheck job, can you point to lava documentation ?